modules/serverless-functions-secrets-configmaps-interactively.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-interactively_{context}"]
+= Adding the configuration interactively
+
+You can manage secrets and ConfigMaps you access from your function using the `func config` command.
```

Comment:
```suggestion
You can manage the secrets and config maps that you access from your function by using the `func config` command.
```

Reply:
Again I think all of these should be `kn func`, `func` afaik is the upstream version but it's a plugin for the `kn` CLI